src/csharp/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -20,18 +20,11 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <DefineConstants>$(DefineConstants);GRPC_CSHARP_SUPPORT_SYSTEM_MEMORY</DefineConstants>
-  </PropertyGroup>
-
   <Import Project="..\Grpc.Core\SourceLink.csproj.include" />
 
   <ItemGroup>
     <PackageReference Include="System.Interactive.Async" Version="3.2.0" />
-  </ItemGroup>
```

Comment:
just wondering, why bumping to 4.5.3?

Reply:
It's not strictly necessary, but it seemed that newer=better.